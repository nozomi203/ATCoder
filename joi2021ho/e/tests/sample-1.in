5 4
3 4 1 1 4
2 5 1 2 1
1 6 3
1 6 4
3 5 1
2 5 9
