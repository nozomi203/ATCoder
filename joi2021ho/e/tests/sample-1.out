-1
29
3
22
